server/src/main/java/com/cloud/api/query/vo/NetworkOfferingJoinVO.java

```diff
@@ -395,4 +398,7 @@ public void setZoneName(String zoneName) {
         this.zoneName = zoneName;
     }
 
+    public String getInternetProtocol() {
+        return internetProtocol;
```

Comment:
@shwstppr 
advice: return internet protocol= "IPv4"  in network offering response by default ?